applications/ecallmgr/src/ecallmgr.hrl

```diff
@@ -309,10 +316,17 @@
                     ,'CHANNEL_DATA', 'CALL_SECURE'
                    ]).
 
+-define(FS_SOFIA_TRANSFER_EVENTS, ['sofia::transferor'
+                                   ,'sofia::transferee'
+                                   ,'sofia::replaced'
+                                   ,'sofia::intercepted'
+                                  ]).
+-define(IS_SOFIA_TRANSFER(N), lists:member(wh_util:to_atom(N, 'true'), ?FS_SOFIA_TRANSFER_EVENTS)).
+
 -define(FS_CUSTOM_EVENTS, ['whistle::noop', 'whistle::masquerade'
                            ,'sofia::transferor', 'sofia::transferee'
-                           ,'sofia::replaced', 'sofia::register'
-                           ,'sofia::intercepted'
+                           ,'sofia::replaced','sofia::intercepted'
+                           ,'sofia::register'
                            ,'conference::maintenance'
                            ,'spandsp::txfaxresult'
                            ,'spandsp::rxfaxresult'
```

Comment:
Please put one atom per line and add a note if order matters, otherwise just sort the lines on alphabetical order